src/lib.rs

```diff
@@ -938,7 +938,7 @@ impl Uuid {
 
 #[cfg(test)]
 mod tests {
-    pub extern crate std;
+    pub use std;
```

Comment:
Looks like this might be causing the build to fail in no-std environments

Reply:
nope... tests module always had the declaration.

I think I just have to revert the change and it should be fine

Reply:
6cbfb3a fixes this